backend/core/listings/gish.json

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
We don't need/want this whole directory now that things are in the database, correct? Ideally I'm thinking that we should convert the test data over to seeds, and then also have tests against the API that validate we're serving correctly. (I'm fine if we do the latter in a separate PR.)

Rather than use the 3 listings here as test data, it would be even better to replace them with the work that Jacky started in #188. That way there's less possibility of collision or confusion with the various real properties. I'm not sure the data in that PR matches our latest thinking in terms of fields and format, but I can't think of a better time to sort it out and then we can close out that PR.

Reply:
I've added seed file that reads that files and inserts them to db. I can change those 3 files to seed.json, would that work?